src/linear_algebra/bilinear_form.lean

```diff
@@ -228,19 +342,14 @@ rfl
 linear_map.to_bilin.symm_symm
 
 @[simp, norm_cast]
-lemma to_linear_map_apply (x : M₂) : ⇑(bilin_form.to_lin B₂ x) = B₂ x := rfl
+lemma bilin_form.to_lin_apply (x : M₂) : ⇑(bilin_form.to_lin B₂ x) = B₂ x := rfl
 
-@[simp]
-lemma map_sum_left {α} (t : finset α) (g : α → M₂) (w : M₂) :
-  B₂ (∑ i in t, g i) w = ∑ i in t, B₂ (g i) w :=
-show bilin_form.to_lin B₂ (∑ i in t, g i) w = ∑ i in t, bilin_form.to_lin B₂ (g i) w,
-by rw [linear_map.map_sum, linear_map.coe_fn_sum, finset.sum_apply]
+/-- Bilinear forms are linearly equivalent to maps with two arguments that are linear in both,
+"arguments-reversed" version. -/
+def bilin_form.to_lin_flip : bilin_form R₂ M₂ ≃ₗ[R₂] M₂ →ₗ[R₂] M₂ →ₗ[R₂] R₂ :=
+bilin_form.flip.trans bilin_form.to_lin
```

Comment:
I'm not very sold on this being a useful definition, since `B.to_lin_flip` is just `B.to_lin.flip`, and rewriting by `linear_equiv.trans_apply` converts it into this form if you really need it.

Reply:
Removed.